ethcore/src/extras.rs

```diff
@@ -32,7 +33,9 @@ pub enum ExtrasIndex {
 	/// Block log blooms index
 	BlockLogBlooms = 3,
 	/// Block blooms index
-	BlocksBlooms = 4
+	BlocksBlooms = 4,
+	/// Block receipts index
+	BlockReceipts
```

Comment:
It should be handled by `rustfmt` automatically. I will look into automating `rustfmt` after finishing my current task.